integration-tests/oauth2-reactive/src/main/java/com/okta/spring/tests/oauth2/reactive/implicit/ReactiveImplicitFlowApplication.java

```diff
@@ -48,8 +49,11 @@ public static void main(String[] args) {
 
         @Bean
         public SecurityWebFilterChain securityWebFilterChain(ServerHttpSecurity http) {
-            return http
-                .authorizeExchange()
+
+            Okta.configureBrowserFriendlyResourceServer401EntryPoint(http);
```

Comment:
@mraible @dogeared I'm not a big fan of this method name, but I'm stuck, I'm guessing there is something more obvious we could do here.

Anyway, these methods basically make a resource server 401 response go from:
```
HTTP/1.1 401
Cache-Control: no-cache, no-store, max-age=0, must-revalidate
Content-Length: 0
Date: Fri, 15 Mar 2019 16:36:18 GMT
Expires: 0
Pragma: no-cache
Set-Cookie: JSESSIONID=some-id; Path=/; HttpOnly
WWW-Authenticate: Bearer
X-Content-Type-Options: nosniff
X-Frame-Options: DENY
X-XSS-Protection: 1; mode=block
```

to:

```
HTTP/1.1 401
Cache-Control: no-cache, no-store, max-age=0, must-revalidate
Content-Length: 17
Content-Type: text/plain;charset=ISO-8859-1
Date: Fri, 15 Mar 2019 16:36:18 GMT
Expires: 0
Pragma: no-cache
Set-Cookie: JSESSIONID=some-id; Path=/; HttpOnly
WWW-Authenticate: Bearer
X-Content-Type-Options: nosniff
X-Frame-Options: DENY
X-XSS-Protection: 1; mode=block

401 UNAUTHORIZED
```

Reply:
`configureResourceServer401ResponseBody`